packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routes.js

```diff
@@ -0,0 +1,7 @@
+import component from './dedicatedCloud-dashboard.component';
```

Comment:
We tends to name file that defined a route in something like: `foo.routing.js`.

What do you think about it?

In comparison:
```sh
$ find . -type f -name "*.routing.js" | wc -l
378

$ find . -type f -name "*.routes.js" | wc -l
106
```